crates/scroll/chainspec/src/lib.rs

```diff
@@ -0,0 +1,610 @@
+//! Scroll-Reth chain specs.
+
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
+    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
+    issue_tracker_base_url = "https://github.com/paradigmxyz/reth/issues/"
+)]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+mod dev;
+mod genesis;
+mod scroll;
+mod scroll_sepolia;
+
+use alloc::{boxed::Box, vec::Vec};
+use alloy_chains::Chain;
+use alloy_consensus::Header;
+use alloy_genesis::Genesis;
+use alloy_primitives::{B256, U256};
+use derive_more::{Constructor, Deref, Display, From, Into};
+pub use dev::SCROLL_DEV;
+pub use genesis::ScrollChainInfo;
+#[cfg(not(feature = "std"))]
+pub(crate) use once_cell::sync::Lazy as LazyLock;
+use reth_chainspec::{
+    BaseFeeParams, ChainSpec, ChainSpecBuilder, DepositContract, EthChainSpec, EthereumHardforks,
+    ForkFilter, ForkId, Hardforks, Head,
+};
+use reth_ethereum_forks::{ChainHardforks, EthereumHardfork, ForkCondition, Hardfork};
+use reth_network_peers::NodeRecord;
+use reth_scroll_forks::ScrollHardforks;
+pub use scroll::SCROLL_MAINNET;
+pub use scroll_sepolia::SCROLL_SEPOLIA;
+#[cfg(feature = "std")]
+pub(crate) use std::sync::LazyLock;
```

Comment:
nit: Can we change the ordering of these dependencies + re-exports. I think a good standard to follow is

1. external dependencies
2. feature gated dependencies
3. `extern *` 
4. local modules + exports

```rust
use alloc::{boxed::Box, vec::Vec};
use alloy_chains::Chain;
use alloy_consensus::Header;
use alloy_genesis::Genesis;
use alloy_primitives::{B256, U256};
use derive_more::{Constructor, Deref, Display, From, Into};
use reth_chainspec::{
    BaseFeeParams, ChainSpec, ChainSpecBuilder, DepositContract, EthChainSpec, EthereumHardforks,
    ForkFilter, ForkId, Hardforks, Head,
};
use reth_ethereum_forks::{ChainHardforks, EthereumHardfork, ForkCondition, Hardfork};
use reth_network_peers::NodeRecord;
use reth_scroll_forks::ScrollHardforks;

#[cfg(not(feature = "std"))]
use once_cell::sync::Lazy as LazyLock;
#[cfg(feature = "std")]
use std::sync::LazyLock;

extern crate alloc;

mod dev;
pub use dev::SCROLL_DEV;

mod genesis;
pub use genesis::ScrollChainInfo;

mod scroll;
pub use scroll::SCROLL_MAINNET;

mod scroll_sepolia;
pub use scroll_sepolia::SCROLL_SEPOLIA;
```

Also I don't think we need the `pub(crate)` specifier before the `LazyLock` dependency?

Reply:
fixed [f47663d](https://github.com/scroll-tech/reth/pull/56/commits/f47663d34a3af6d6b39b879f7c7eb7711f6f566b)